src/libraries/System.ServiceModel.Syndication/tests/System/ServiceModel/Syndication/AtomPub10ServiceDocumentFormatterTests.cs

```diff
@@ -490,7 +489,7 @@ public void ReadFrom_FullDocument_ReturnsExpected()
                 Assert.Equal("category_scheme", secondDocumentSecondCategory.Scheme);
                 Assert.Equal("category_label", secondDocumentSecondCategory.Label);
 
-                InlineCategoriesDocument thirdDocument = Assert.IsType<InlineCategoriesDocument>(secondCollection.Categories[2]);
+                Assert.IsType<InlineCategoriesDocument>(secondCollection.Categories[2]);
```

Comment:
Doesn't this look like a copy/paste from line 474?  Which was for a "category". When these few lines here should actually be checking the third "document" similar to the first document at 451 or the fourth document at 500 or the fifth document at 512? I don't think this change here will break anything of course, but it looks like the intent of the test writer was probably to verify the third document rather than re-verify the second collection.

Reply:
It also looks like this mistaken code was copied down to line 668 as well, so the same type of change would be needed there.

Reply:
Yeah, now that you mention it, it does look that way. I've pushed a commit that adds the document back and fixes up the asserts (asserting the values on all properties of the object). Let me know if any of them look suspicious to you. Thanks!